nautobot/extras/api/serializers.py

```diff
@@ -427,29 +427,28 @@ class JobVariableSerializer(serializers.Serializer):
 
 
 #
-# Job Results
+# Scheduled Jobs
 #
 
 
-class JobResultSerializer(CustomFieldModelSerializerMixin, BaseModelSerializer):
-    status = ChoiceField(choices=JobResultStatusChoices, read_only=True)
-    obj_type = ContentTypeField(read_only=True)
+class ScheduledJobSerializer(BaseModelSerializer):
+    # start_time = serializers.DateTimeField(format=None, required=False)
```

Comment:
TODO: why was this commented out?

Reply:
Is this still commented out for some reason?